site2/docs/getting-started-standalone.md

```diff
@@ -158,78 +166,55 @@ If you have started Pulsar successfully, you will see `INFO`-level log messages
 
-## Run Pulsar Standalone in Docker
+## Use Pulsar standalone
 
-Alternatively, you can run a Pulsar standalone locally in docker.
+Pulsar provides a CLI tool called [`pulsar-client`](reference-cli-tools.md#pulsar-client). The pulsar-client tool enables you to consume and produce messages to a Pulsar topic in a running cluster. 
+
+### Consume a message
+
+The following command consumes a message with the subscription name `first-subscription` to the `my-topic` topic:
 
 ```bash
-docker run -it -p 80:80 -p 8080:8080 -p 6650:6650 apachepulsar/pulsar-standalone
+$ bin/pulsar-client consume my-topic -s "first-subscription"
 ```
 
-The command forwards the following port to localhost:
+If the message has been successfully consumed, you will see a confirmation like the following in the `pulsar-client` logs:
 
-- 80: the port for Pulsar dashboard
-- 8080: the HTTP service URL for Pulsar service
-- 6650: the binary protocol service URL for Pulsar service
+```
+09:56:55.566 [pulsar-client-io-1-1] INFO  org.apache.pulsar.client.impl.MultiTopicsConsumerImpl - [TopicsConsumerFakeTopicNamee2df9] [first-subscription] Success subscribe new topic my-topic in topics consumer, partitions: 4, allTopicPartitionsNumber: 4
+```
 
-After the docker container is running, you can access the dashboard at http://localhost.
+> #### Tip
+>  
+> As you have noticed that we do not explicitly create the `my-topic` topic, to which we consume the message. When you consume a message to a topic that does not yet exist, Pulsar creates that topic for you automatically. Producing a message to a topic that does not exist will automatically create that topic for you as well.
 
-## Test Your Cluster Setup
+### Produce a message
 
-Pulsar provides a CLI tool called [`pulsar-client`](reference-cli-tools.md#pulsar-client). The pulsar-client enables you to send messages to a Pulsar topic in a running cluster. The following command sends a message saying `hello-pulsar` to the `my-topic` topic:
+The following command produces a message saying `hello-pulsar` to the `my-topic` topic:
 
 ```bash
 $ bin/pulsar-client produce my-topic --messages "hello-pulsar"
 ```
 
-If the message has been successfully published to the topic, you will see a confirmation like this in the `pulsar-client` logs:
+If the message has been successfully published to the topic, you will see a confirmation like the following in the `pulsar-client` logs:
 
 ```
 13:09:39.356 [main] INFO  org.apache.pulsar.client.cli.PulsarClientTool - 1 messages successfully produced
 ```
 
+## Stop Pulsar standalone
 
-> #### No need to create new topics explicitly
-> As you have noticed that we do not explicitly create the `my-topic` topic, to which we sent the `hello-pulsar` message. When you write a message to a topic that does not yet exist, Pulsar creates that topic for you automatically.
+Use `ctrl + c` to stop a local standalone Pulsar.
```

Comment:
We can adopt either of the following two options:
1. Press Ctrl+C to ... (from IBM style guides)
2. Press CTRL+C to ... (from Microsoft guides)
- When you write about two or more keys that are used in combination to perform a function, use the plus sign (+) to separate the key names. Do not insert a space before or after the plus sign.

Reply:
Thanks for your reminding, I've incorporated your comment. 

Option 1 is preferred because the following style guides choose **Ctrl+C** and **Ctrl+Alt+Delete**:

- **DITA Best Practices**
- **Chicago Manual of Style** 
- **Microsoft Manual of Style** 
- **Technical Writing Made Easier**
- **Developing Quality Technical Information**


and **Microsoft Manual of Style for Technical Publication** goes for **CTRL+C** and **CTRL+ALT+DELETE**

while **HP Writing Style Guide**  selects **Ctrl+c** and **Ctrl+Alt+Delete**